### PR TITLE
Break up scopes docs

### PR DESCRIPTION
**Pull request by khorne3**
* New Index: https://auth0-docs-content-pr-6490.herokuapp.com/docs/scopes/current
* API Scopes: https://auth0-docs-content-pr-6490.herokuapp.com/docs/scopes/current/api-scopes
* Custom Claims: https://auth0-docs-content-pr-6490.herokuapp.com/docs/scopes/current/custom-claims
* OIDC Scopes: https://auth0-docs-content-pr-6490.herokuapp.com/docs/scopes/current/oidc-scopes

### CODE REVIEW

**Review by tamigoodall — DISMISSED** [review]

**Comment by khorne3** [comment]
Whoops, forgot to tag this! Not quite done yet :)

File: articles/scopes/current/api-scopes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+useCase:
+  - development
+---
+# API Scopes
+
+API scopes allow you to define the API data accessible to your applications. 
+
+## Background
+
+When you [create an API in Auth0](/apis), you'll need to define one scope for each API represented and action. For example, if you want to `read` and `delete` contact information, you would create two scopes: `read:contacts` and `delete:contacts`.
```
Comment (jeffreylees) [inline articles/scopes/current/api-scopes.md]: This sentence reads a little unclear `define one scope for each API represented and action`. Can it be rephrased?

File: articles/scopes/current/api-scopes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Once you create an API and define the scopes, the applications can request these defined permissions when they initiate an authorization flow and include them in the Access Token as part of the scope request parameter.
+
+If you wanted to expand [our example](#example-asking-for-standard-claims) to include also the `read:contacts` permission, then you would using something like the following sample URL to initiate the authentication flow using the Implicit grant:
```
Comment (jeffreylees) [inline articles/scopes/current/api-scopes.md]: This link doesn't go anywhere?

File: articles/scopes/current/api-scopes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  audience=YOUR_API_AUDIENCE&
+  scope=openid%20profile%20email%20read:contacts&
+  response_type=id_token%20token&
+  client_id=${account.clientId}&
+  redirect_uri=${account.callback}&
+  nonce=YOUR_CRYPTOGRAPHIC_NONCE
+  state=YOUR_OPAQUE_VALUE
+```
+
+Note the differences between the two examples. In the latest, we want to get an Access Token, that will allow us to access the API, with the rights to do specific actions. To do so, we changed two parameters and added a new one:
```
Comment (jeffreylees) [inline articles/scopes/current/api-scopes.md]: I think this ties in to the above missing example, can't note the difference if there aren't two.

File: articles/scopes/current/api-scopes.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+- `audience`: New parameter added for this example. Its value is the unique identifier of the API we want to get access to.
+
+- `scope`: We appended the value `read:contacts`. This denotes the rights that we want to be granted at the API (in this case, read contact information).
+
+- `response_type`: We appended the value `token`. This tells the Authorization Server (Auth0 in our case) to issue an Access Token as well, not only an ID Token. The Access Token will be sent to the API as credentials.
+
+## Define Scopes Using the Dashboard
```
Comment (jeffreylees) [inline articles/scopes/current/api-scopes.md]: Sentence casing for H2s throughout

File: articles/scopes/current/api-scopes.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+![API Scopes](/media/articles/scopes/api-scopes.png)
+
+## Limiting API Scopes being Issued
+
+An application can request any scope and the user will be prompted to approve those scopes during the authorization flow. This may not be a desirable situation, as you may want to limit the scopes based on, for example, the permissions (or role) of a user.
+
+You can make use of the [Authorization Extension](/extensions/authorization-extension) in conjunction with a custom [Rule](/rules) to ensure that scopes are granted based on the permissions of a user.
+
+This approach is discussed in more depth in some of our [Architecture Scenarios](/architecture-scenarios). Specifically, you can review the entire [Configure the Authorization Extension](/architecture-scenarios/application/spa-api/part-2#configure-the-authorization-extension) section of our SPA+API Architecture Scenario which demonstrates how to configure the Authorization Extension, and also create a custom Rule which will ensure scopes are granted based on the permissions of a user.
```
Comment (jeffreylees) [inline articles/scopes/current/api-scopes.md]: Configure the Authorization Extension link 404s

File: articles/scopes/current/oidc-scopes.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+# OpenID Connect Scopes
+
+OpenID Connect (OIDC) is an authentication protocol that sits on top of OAuth2, and allows the application to verify the identity of the users and obtain basic profile information about them in a interoperable way. This information can be returned in the ID Token and/or in the response from [the /userinfo endpoint](/api/authentication#get-user-info) (depending on the type of request).
+
+The basic (and required) scope for OpenID Connect is the `openid` scope. This scope represents the intent of the application to use the OIDC protocol to verify the identity of the user.
+
+In OpenID Connect (OIDC), we have the notion of __claims__. There are two types of claims:
+
+* [Standard](#standard-claims) (which means that they meet OIDC specification)
+* [Custom](#custom-claims)
```
Comment (jeffreylees) [inline articles/scopes/current/oidc-scopes.md]: This goes nowhere - does it need to be `(/scopes/current/custom-claims)` now?

**Review by jeffreylees — CHANGES_REQUESTED** [review]

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**